whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -362,6 +367,10 @@ private RdapObject mapCommonConformances(final RdapObject rdapResponse) {
         return rdapResponse;
     }
 
+    private void includeRirSearchConformance(final RdapObject rdapResponse, final String requestUrl) {
```

Comment:
Why this is in rdapObjectMapper? This is specific to relation RDAP search so the way it should happen is rdaprelationService should have dependency on rdapObjectMapper and that RdapRelationService should call rdapObjectMapper get the object and then add RirSearchConformance , this way we are not mixing normal rdapObjectMapper code with relationObjectMapper code.